fontforge/nowakowskittfinstr.c

```diff
@@ -26,20 +26,23 @@
  * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+
+#include <fontforge-config.h>
+
 #include "autohint.h"
 #include "dumppfa.h"
 #include "fontforgevw.h"
 #include "mem.h"
-#include <math.h>
-#include <utype.h>
-
-#include "ttf.h"
 #include "splinefont.h"
-#include "stemdb.h"
 #include "splineutil.h"
 #include "splineutil2.h"
 #include "stemdb.h"
+#include "stemdb.h"
```

Comment:
Duplicate before but not clear it's needed.

Reply:
Yeah, so I sorted the includes but didn't search for duplicates. I've reran, searching for duplicates, and removed a couple more, in addition to this instance